egov-pdf/LOCALSETUP.md

```diff
@@ -0,0 +1,20 @@
+# Local Setup
+
+To setup the egov-pdf service in your local system, clone the [utilities Service repository](https://github.com/egovernments/utilities).
+
+## Dependencies
+
+### Infra Dependency
+
+- [ ] Postgres DB
+- [ ] Redis
+- [ ] Elasticsearch
+- [ ] Kafka
+  - [ ] Consumer
+  - [ ] Producer
+
+## Running Locally
+
+- Provide host url in `HOST` property in `config.js`
```

Comment:
In readme, we are saying services dependencies as below

- egov-mdms-service
- tl-service
- Property-service
- Pdf-service
- User-service
- Collection service
- Billing-service
- Workflow-service

But here there is no talk about any configuration? Just confirm this once

Reply:
Added the list of properties which has to be updated before running